### DISCUSSION TITLE
Columns have strings with the same length but different width.

### DISCUSSION THREAD

**Discussion by meotimdihia** [dbeaver/dbeaver#17812] [Q&A] [answered]
Anyone knows how to fix this problem. It looks so bad.
The screenshot is on macOS.
<img width="121" alt="image" src="https://user-images.githubusercontent.com/300961/193983928-ef45a34a-9077-4639-90d2-40aa7fc66827.png">

**Comment by SubOptimal** [comment, marked as answer]
If you want all digits to be displayed with the same width, you need to configure a monospaced font for the result grid.

- go to settings `User Interface > Appearance > Colors and Fonts` and search for `grid font`
![image](https://user-images.githubusercontent.com/653288/193991673-f3c1d83e-26bb-4715-ad42-6e776378cfc3.png)
- click on edit and select a monospaced font, for demonstration `Fira Code Regular 11` has been selected (check the example text in the "Preview" area)
![image](https://user-images.githubusercontent.com/653288/193991868-1d0a761f-0765-4e71-a459-1c4b32bd4de1.png)
- the digits in the result grid now have the same width
![image](https://user-images.githubusercontent.com/653288/193992061-4f72d6a3-bd86-4707-bd6c-6dbb08fa4da5.png)

**Reply by meotimdihia** [reply]
Thanks, I think we should set the default font to Mono.
